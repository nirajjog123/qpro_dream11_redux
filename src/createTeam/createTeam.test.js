import React from "react";
import "jest-dom/extend-expect";
import Adapter from "enzyme-adapter-react-16";
import { shallow, configure } from "enzyme";
import sinon from "sinon";

import ConnectedApp, { createTeam } from "./createTeam";

configure({ adapter: new Adapter() });
const spy = sinon.spy();

function setup() {
  const props = {
    handleTeamName: jest.fn()
  };


  const enzymeWrapper = shallow(<createTeam {...props} />);

  return {
    props,
    enzymeWrapper
  };
}

describe("<CreateTeam />", () => {
  const { enzymeWrapper, props } = setup();

  it("it should render properly ", () => {
    expect(enzymeWrapper).toMatchSnapshot();
  });

//   it("saveTeam should be called atleast once", () => {
//     const onButtonClick = spy;

//     const inputD = shallow(<createTeam onButtonClick={onButtonClick} />);

//     inputD.find('input').simulate('click');
//     //expect(props.handleTeamName).toBeCalled();

//     expect(enzymeWrapper).toMatchSnapshot();
//   });
});
