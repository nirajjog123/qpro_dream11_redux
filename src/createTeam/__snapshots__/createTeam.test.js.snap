// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateTeam /> it should render properly  1`] = `ShallowWrapper {}`;
